runtime/src/bank.rs

```diff
@@ -1155,16 +1155,23 @@ struct VoteWithStakeDelegations {
     delegations: Vec<(Pubkey, StakeAccount<()>)>,
 }
 
+type VoteWithStakeDelegationsMap = DashMap<Pubkey, VoteWithStakeDelegations>;
+
+type InvalidCacheKeyMap = DashMap<Pubkey, InvalidCacheEntryReason>;
+
 struct LoadVoteAndStakeAccountsResult {
-    vote_with_stake_delegations_map: DashMap<Pubkey, VoteWithStakeDelegations>,
-    invalid_stake_keys: DashMap<Pubkey, InvalidCacheEntryReason>,
-    invalid_vote_keys: DashMap<Pubkey, InvalidCacheEntryReason>,
+    vote_with_stake_delegations_map: VoteWithStakeDelegationsMap,
+    invalid_stake_keys: InvalidCacheKeyMap,
+    invalid_vote_keys: InvalidCacheKeyMap,
```

Comment:
What is the value of this type aliases here?
For someone reading the code, a `DashMap<Pubkey, VoteWithStakeDelegations>` is much more explicit and informative what type we are dealing with.
Whereas `VoteWithStakeDelegationsMap` is just obscuring that, and you have to jump elsewhere to find out what that is and mentally always do this back-and-forth conversion to the actual type you are using.